src/admin/components/ResourceTabs/InformationTab/components/ReportInformationAside/HighLevelMetrics.tsx

```diff
@@ -15,7 +15,8 @@ const HighLevelMetics: FC = () => {
   };
 
   const workdaysType = resource === "projectReport" ? "Project" : resource === "siteReport" ? "Site" : null;
-
+  const isTerrafund = record.framework_key === Framework.TF || record.framework_key === Framework.TF_LANDSCAPES;
+  const isSiteReport = resource === "siteReport" && isTerrafund;
```

Comment:
I think including `isTerrafund` here isn't correct. For instance, it looks like the code under the block on line 44 will never display because it's wrapped in a `!isSiteReport` `When` block, but then its declaring that its children should only display when it _is_ TF.